Specifications/Language/3_Expressions/ConditionalLoops.md

```diff
@@ -64,6 +62,6 @@ while index < Length(arr) && item < 0 {
 The statement block is run as long as the condition evaluates to `true`.
 
 > [!NOTE]
-> Due to the challenge they pose for execution, we discourage the use of loops that break based on a condition and hence do not support `while` loops within operations. The use of `while` loops within operations may be considered in the future, with the restriction that the condition cannot depend on the outcome of a quantum measurement. 
+> Due to the challenge they pose for execution, we discourage the use of loops that break based on a condition and hence do not support `while` loops within operations. The use of `while` loops within operations may be considered in the future, with the restriction that the condition cannot depend on the outcome of a quantum measurement.
```

Comment:
Ah, here's the note about it :). Since while loops are now allowed in operations, this note can just be removed.

Reply:
removed